app/components/ui/Breadcrumbs.vue

```
@@ -0,0 +1,62 @@
+useHead({
+  script: computed(() => 
+    jsonLd.value ? [{
+      type: 'application/ld+json',
+      innerHTML: JSON.stringify(jsonLd.value),
+    }] : []
+  ),
+})
+</script>
+
+<template>
+  <nav 
```

ESLint

[error] Trailing spaces not allowed.

(style/no-trailing-spaces)